### DISCUSSION TITLE
Do you support graphql?

### DISCUSSION THREAD

**Discussion by adiazarya100** [DamianOsipiuk/vue-query#122] [Q&A] [answered]
I'm using graphql and I wonder if I can use it with vue-query.
react-query supports graphql also with graphql-code-generator...

tnx

**Comment by DamianOsipiuk** [comment, marked as answer]
Hi,
Since `vue-query` uses `react-query/core` under the hood almost all of the features of react-query are supported, including protocol agnostic data fetching.
So yes you can use graphql.

As of `graphql-code-generator` you could probably modify the `react-query` one to use different imports as signature of hooks are almost exactly the same.
